editor/Storyboarding/EditorStoryboardLayer.cs

```diff
@@ -72,9 +72,10 @@ public bool DiffSpecific
         protected void RaiseChanged(string propertyName)
             => EventHelper.InvokeStrict(() => OnChanged, d => ((ChangedHandler)d)(this, new ChangedEventArgs(propertyName)));
 
-        public EditorStoryboardLayer(string identifier, Effect effect) : base(identifier)
+        public EditorStoryboardLayer(string identifier, Effect effect, OsbLayer osbLayer = OsbLayer.Background) : base(identifier)
         {
             this.effect = effect;
+            this.osbLayer = Enum.IsDefined(typeof(OsbLayer), (int)osbLayer) ? osbLayer : OsbLayer.Background;
```

Comment:
Was intended as a failsafe in the event that an undefined enum value was passed into the argument (i.e. out-of-range from the defined four layers). Previously, by only being able to change layer via the UI, there was no possibility of this happening.

Though honestly, this shouldn't happen unless someone was deliberately casting an ``int`` violating this into an ``OsbLayer``. If anyone is playing with the direct ``int`` values instead of the ``enum`` type for this, they're doing something incredibly wrong.

I find that the line is a bit ugly. Do you rather think it's unnecessary, or there should be a more elegant way of failsafing to ``OsbLayer.Background``?